src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/layer-panels.component.ts

```diff
@@ -44,8 +44,8 @@ export class LayerPanelsComponent implements OnInit {
         const isVirtualContent = this.content.isVirtual();
         const schemas = this.showContentSchema && isVirtualContent ? [this.content.contentSchemaId] : [];
 
-        if (contentSchema.layerSchemaIds) {
-          schemas.push(...contentSchema.layerSchemaIds);
+        if (contentSchema.filteredLayerSchemaIds) {
```

Comment:
It should still be possible to show XMP / Exif layers. Maybe pass excludedSchemaIds to layer-panels component?

Reply:
Done